### ISSUE TITLE
After installing VS Code updates, when running npm scripts, I get stdin is not a tty

### ISSUE THREAD

**Issue by budarin** [microsoft/vscode#139025]
Issue Type: <b>Bug</b>

when starting the IDE today, the terminal I preset by me (Bash) was replaced to Microsoft shell, and after that, when running npm script commands, I get the message "stdin is not a tty"

changing the terminal to the previously set Bash did not lead to anything - the error is still present

I started executing the script by commands - everything works
I noticed that if to run commands for execution in parallel via & - this error occurs

Please help - the work has stopped (

VS Code version: Code 1.63.0 (7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e, 2021-12-07T06:26:56.179Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD FX(tm)-6300 Six-Core Processor              (6 x 3512)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|23.90GB (11.87GB free)|
|Process Argv|--disable-extensions --crash-reporter-id db8247ca-9b13-487a-a022-6636eb0e2750|
|Screen Reader|no|
|VM|0%|
</details>Extensions disabled<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30384386
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsaa593:30376534
pythonvs932:30410667
vscop804cf:30404767
vscop940:30404999
vsrem710cf:30405999
vscexrecpromp3t1:30407762

```

</details>

</details><details>
<summary>Here is my custom terminal settings</summary>

```json
{
    "terminal.integrated.cursorBlinking": true,
    "terminal.integrated.enableBell": true,
    "terminal.integrated.fontFamily": "Consolas, 'Courier New', monospace",
    "terminal.integrated.minimumContrastRatio": 4.5,
    "terminal.integrated.defaultProfile.windows": "Git Bash",
    "terminal.integrated.profiles.windows": {
        "Bash": {
        "path": "C:\\Program Files\\Git\\bin\\bash.exe",
        "args": []
      }
    },
    "terminal.integrated.scrollback": 500000,
    "terminal.integrated.enablePersistentSessions": false,
    "terminal.external.windowsExec": "'C:\\Program Files\\Git\\bin\\bash.exe`"
  }
  ```

 </details>
 
P.S.:  script has no any `docker -it ` commands

**Comment by Tyriar** [comment]
Please provide a screenshot or [a gif](https://gifcap.dev/) illustrating the problem (attaching images won't work via email, you need to go to github.com). See [this wiki page](https://github.com/microsoft/vscode/wiki/Terminal-Issues) for more general info on how to report actionable terminal issues.

/label "needs more info" terminal

**Comment by budarin** [comment]
![image](https://user-images.githubusercontent.com/8055157/146052357-bc1f251b-e3b2-4c2c-bcc2-ff9d24da9df3.png)

**Comment by Tyriar** [comment]
@budarin to clarify, if you run `yarn run start-dev` in a regular pwsh terminal (not a task), does it work?

**Comment by budarin** [comment]
![image](https://user-images.githubusercontent.com/8055157/146428348-fcbda9ba-b812-4904-916e-e82b9a268a63.png)

**Comment by budarin** [comment]
Could this bug be due to the installation of a new version of Git?

**Comment by Tyriar** [comment]
Oh, you're running Windows and trying to run `sh`? Yeah, this seems like some issue with whatever shell you're trying to run `sh` within (git bash?). I'd recommend trying to get `./scripts/start-dev.sh` running in whatever shell you usually use, it's not going to work in powershell.
